src/Material/Tooltip.elm

```diff
@@ -0,0 +1,523 @@
+module Material.Tooltip
+  exposing
+    ( Model
+    , defaultModel
+    , Msg(..)
+    , update
+    , view
+    , render
+    , DOMState
+    , left
+    , right
+    , top
+    , bottom
+    , default
+    , large
+    , Property
+    , onMouseEnter

```

Comment:
`onMouseEnter` and `onMouseLeave` are needed if you want to have a tooltip for a regular `Html m` element.